tgui/packages/tgui/styles/interfaces/HellishRunes.scss

```diff
@@ -0,0 +1,22 @@
+@use '../colors.scss';
+@use '../base.scss';
+
+$bg-map: colors.$bg-map !default;
+
+.demon__demonrune {
```

Comment:
Wouldn't it be better to call this class the same way as the base filename? Or even further, the same way as the interface you coded this for.

Reply:
Aha, so like `HellishRunes__demonrune?`